src/macos-symbol-layer.ts

```diff
@@ -16,7 +16,7 @@ export const macosSymbolLayerKeys: Record<string, string> = {
   ['º']: '0',
   ['–']: '-',
   ['≠']: '=',
-  ['`']: '~',
+  ['`']: '`',
```

Comment:
We should be able to just delete this line, since the default is to use the key itself:

https://github.com/github/hotkey/blob/bc70e3d21e4e7a27544643c43b125c8ee0672004/src/hotkey.ts#L50

```suggestion
```